numpy/core/src/multiarray/shape.c

```diff
@@ -87,7 +87,14 @@ PyArray_Resize(PyArrayObject *self, PyArray_Dims *newshape, int refcheck,
         }
 
         if (refcheck) {
+#ifdef PYPY_VERSION
+            PyErr_SetString(PyExc_ValueError,
+                    "cannot resize an array on PyPy."\
+                    "  Use the resize function");
```

Comment:
A bit of making code prettier never hurts, but... I don't mind not mentioning the keyword, though in the other error message things are bit clearer.